topo/node/ceos/ceos.go

```diff
@@ -45,6 +45,10 @@ func defaultSpawner(command string, timeout time.Duration, opts ...expect.Option
 	return expect.Spawn(command, timeout, opts...)
 }
 
+var (
+	mySecond = time.Second
```

Comment:
Nit: Personally I think its more clear when stubbing for testing to call this timeSecond instead of mySecond to be clear this is a stub of the duration time.Second

Reply:
agreed